pycbc/distributions/sky_location.py

```diff
@@ -31,5 +34,68 @@ class UniformSky(angular.UniformSolidAngle):
     _default_polar_angle = 'dec'
     _default_azimuthal_angle = 'ra'
 
+class FisherDist():
+    """A distribution that returns a random (ra, dec) angle drawn from the
+    Fisher distribution. Assume that the concentration parameter (kappa)
+    is large so that we can use a Rayleigh distribution about the north
+    pole and rotate it to be centered at the (ra, dec) coordinate mu.
 
-__all__ = ['UniformSky']
+    Assume kappa = 1 / sigma**2
+
+    As in UniformSky, the declination (dec) varies from pi/2 to-pi/2
+    and right ascension (ra) varies from 0 to 2pi. And the angles
+    should be provided in (ra,dec) format in radians (mu_radians=True),
+    rather than factors of pi, or in degrees (mu_radians=False).
```

Comment:
Which units should `kappa` be given in? Please document this in the docstring as well. Should the option `mu_radians` also affect the units of `kappa`?

Reply:
The unit is now included in the docstring. It should be in steradians as well. Yes, mu_radians should be affecting units of kappa not directly but via rotate_euler.